doc/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+cd $(dirname $(realpath $0))
+
+cp ../README.md index.md
+
+make clean
+make
+
+# fix internal doc links
+sed -i 's/https:\/\/github\.com\/doc\/\(.*\)\.md/\1.html/g' *.html
```

Comment:
Is it edown that generates the github.com links? We should link to hexdocs instead.

Reply:
It is replacing the whole github url with just the filename.

Actually might not be needed for hex_core's docs.